python-threatexchange/threatexchange/cli/main.py

```diff
@@ -141,8 +143,82 @@ def _get_fb_tx_app_token(config: CLiConfig) -> t.Optional[str]:
 def _get_stopncii_tokens(
     config: CLiConfig,
 ) -> t.Tuple[t.Optional[str], t.Optional[str]]:
-    """Get the API keys for StopNCII from the config"""
-    return None, None  # TODO
+    """
+    Get the API key from a variety of fallback sources
+
+    Examples might be environment, files, etc
+    """
```

Comment:
As I was writing this, I wondered whether I shouldn't add a place for this on the signal exchange API itself, since otherwise I keep adding these on individual classes. Getting the default from environment/file seems simple enough to make default behavior in the constructor

Reply:
Why have fallbacks at all? What prompted this feature?

Reply:
Offline discussion: Good CLI practice reasons - using the config to create a password will cause it to be stored visibly and in plaintext in bash history, which is an antipattern. Providing alternatives gives security-conscious users an alternative.